### PR TITLE
Fix dygraph show style

### PR DESCRIPTION
**Pull request by junjun315**
Fix dygraph show style for FluidDoc

### CODE REVIEW

**Review by chengduoZH — DISMISSED** [review]

**Review by xsrobin — DISMISSED** [review]

**Review by xsrobin — DISMISSED** [review]

**Review by luotao1 — DISMISSED** [review]

File: python/paddle/fluid/dygraph/nn.py [hunk 1]
```diff
@@ -681,27 +676,30 @@ def forward(self, input):
...
+            the pool stride size will be a square of an int.
         pool_padding (int|list|tuple): The pool padding size. If pool padding size is a tuple,
             it must contain two integers, (pool_padding_on_Height, pool_padding_on_Width).
             Otherwise, the pool padding size will be a square of an int.
-        global_pooling (bool): ${global_pooling_comment}
-        use_cudnn (bool): ${use_cudnn_comment}
-        ceil_mode (bool): ${ceil_mode_comment}
-        name (str|None): A name for this layer(optional). If set None, the
-                        layer will be named automatically.
+        global_pooling (bool): (bool, default false) Whether to use the global pooling. If global_pooling = true,
```
Comment (xsrobin) [inline python/paddle/fluid/dygraph/nn.py]: 多了bool

File: python/paddle/fluid/dygraph/nn.py [hunk 2]
```diff
@@ -681,27 +676,30 @@ def forward(self, input):
...
+            kernel size and paddings will be ignored
+        use_cudnn (bool): (bool, default True) Onlyceil_mode (bool) - (bool, default false) Whether to use the ceil
```
Comment (xsrobin) [inline python/paddle/fluid/dygraph/nn.py]: 多了bool，ceil_mode另起一行

File: python/paddle/fluid/dygraph/nn.py [hunk 3]
```diff
@@ -681,27 +676,30 @@ def forward(self, input):
...
+            function to calculate output height and width. False is the default.
```
Comment (xsrobin) [inline python/paddle/fluid/dygraph/nn.py]: 删除False is the default.

**Review by xsrobin — COMMENTED** [review]
as comments

File: python/paddle/fluid/dygraph/nn.py (hunk 3 above)
Comment (junjun315) [reply python/paddle/fluid/dygraph/nn.py]: done

File: python/paddle/fluid/dygraph/nn.py (hunk 2 above)
Comment (junjun315) [reply python/paddle/fluid/dygraph/nn.py]: done

File: python/paddle/fluid/dygraph/nn.py (hunk 1 above)
Comment (junjun315) [reply python/paddle/fluid/dygraph/nn.py]: done

**Review by xsrobin — DISMISSED** [review]

**Review by xsrobin — APPROVED** [review]

**Review by luotao1 — APPROVED** [review]

**Pull request merged**